base_external_dbsource/models/base_external_dbsource.py

```diff
@@ -62,58 +63,60 @@
     _logger.info('Oracle libraries not available. Please install "cx_Oracle"\
                  python package.')
 
-import psycopg2
 CONNECTORS.append(('postgresql', 'PostgreSQL'))
 
 
-class base_external_dbsource(orm.Model):
+class BaseExternalDbsource(models.Model):
     _name = "base.external.dbsource"
     _description = 'External Database Sources'
-    _columns = {
-        'name': fields.char('Datasource name', required=True, size=64),
-        'conn_string': fields.text('Connection string', help="""
-Sample connection strings:
-- Microsoft SQL Server:
-  mssql+pymssql://username:%s@server:port/dbname?charset=utf8
-- MySQL: mysql://user:%s@server:port/dbname
-- ODBC: DRIVER={FreeTDS};SERVER=server.address;Database=mydb;UID=sa
-- ORACLE: username/%s@//server.address:port/instance
-- PostgreSQL:
-  dbname='template1' user='dbuser' host='localhost' port='5432' password=%s
-- SQLite: sqlite:///test.db
-"""),
-        'password': fields.char('Password', size=40),
-        'connector': fields.selection(CONNECTORS, 'Connector',
-                                      required=True,
-                                      help="If a connector is missing from the\
+    name = fields.Char('Datasource name', required=True, size=64)
+    conn_string = fields.Text('Connection string', help="""
+    Sample connection strings:
+    - Microsoft SQL Server:
+      mssql+pymssql://username:%s@server:port/dbname?charset=utf8
+    - MySQL: mysql://user:%s@server:port/dbname
+    - ODBC: DRIVER={FreeTDS};SERVER=server.address;Database=mydb;UID=sa
+    - ORACLE: username/%s@//server.address:port/instance
+    - PostgreSQL:
+        dbname='template1' user='dbuser' host='localhost' port='5432' \
+        password=%s
+    - SQLite: sqlite:///test.db
+    """)
```

Comment:
@naousse Can you add this `help` string as the placeholder as well?

Otherwise :+1: